.env.development.sample

```
@@ -1,5 +1,31 @@
+#
+# PUBLIC values are safe to commit/share: every NEXT_PUBLIC_* var ships in the
+# browser bundle, and the Supabase key here is the *publishable* key, gated by
+# Row-Level Security. SECRET values must never be committed (they're gitignored
+# via `.env.local`).
+# ─────────────────────────────────────────────────────────────────────────────
 
-NEXT_PUBLIC_SUPABASE_URL="https://<project-ref>.supabase.co"
-NEXT_PUBLIC_SUPABASE_PUBLISHABLE_KEY="sb_publishable_..."
+# ── Public · required for local dev ──────────────────────────────────────────
+# The app talks to Supabase over its REST API using these two.
+NEXT_PUBLIC_SUPABASE_URL="https://lthqyqlislwikgoxmttn.supabase.co"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)